CRM/Contribute/Import/Parser/Contribution.php

```diff
@@ -235,7 +235,7 @@ public function getMappedRow(array $values): array {
           $params[$entity] = [];
           $params[$entity]['contact_type'] = $this->getContactTypeForEntity($entity) ?: $this->getContactType();
         }
-        $params[$entity][$this->getFieldMetadata($mappedField['name'])['name']] = $this->getTransformedFieldValue($mappedField['name'], $fieldValue);
+        $params[$entity][$mappedField['name'] === 'contribution_campaign_id' ? $mappedField['name'] : $this->getFieldMetadata($mappedField['name'])['name']] = $this->getTransformedFieldValue($mappedField['name'], $fieldValue);
```

Comment:
@eileenmcnaughton I found that I had to do this otherwise I was getting undefined index campaign_id notices in the test

Reply:
@seamuslee001 can you add to this line https://github.com/civicrm/civicrm-core/blob/99ba407d2c44c16eb4894af0407bbec768dcd021/CRM/Contribute/Import/Parser/Contribution.php#L794 & remove this change?